docs/DeveloperGuide.adoc

```diff
@@ -797,73 +873,414 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |Exco member |remove a particular tag/group | remove redundancies or remove irrelevant tags
+
+|`* * *` |Exco member |remove a member from a particular group | keep the App up-to-date, and also take into account students that opt-out or graduate
 
-|`* * *` |user |add a new person |
+|`* * *` |Exco member |add a task for certain members | assign members to do a task
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |Exco member |view the status of a task | see the progress made towards completing the task
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |Exco member |remove a task | delete tasks that have been completed
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |Exco member |create a poll | get the opinions of the club members
+
+|`* * *` |Exco member who created a poll |view results of the poll | see the opinions of the members
+
+|`* * *` |Member |edit my contact details |keep my contact information updated
+
+|`* * *` |Member |undo my command |reverse any undesirable commands or mistakes
+
+|`* * *` |Member |redo a command |return to the state before I undid a command
+
+|`* * *` |Member |select a member |view the member's details
+
+|`* * *` |Member |find a member by name |locate details of members without having to go through the entire list
+
+|`* * *` |Member |find members by a specified field |locate details of members without having to go through the entire list
+
+|`* * *` |Member |get notifications for new tasks |be aware of new tasks assigned to me
+
+|`* * *` |Member |add a display picture |be identified by others
+
+|`* * *` |Member |add multiple <<entry,entries>> for <<attribute,attributes>> |provide all alternative contact details to others
+
+|`* * *` |Member |logout of the application |maintain integrity of my data and actions
+
+|`* * *` |Member |login to my account |access my data
+
+|`* * *` |Member |vote in a poll |provide my opinions
+
+|`* * *` |Member |remove a task assigned to me |focus on the yet to be done tasks
+
+|`* * *` |Busy member |be able to directly email members using my default mail client |save time and reduce errors by not needing to add recipients myself
+
+|`* * *` |Member |export members' info |so that I can share the details easily
+
+|`* * *` |Exco member |import data |add members' info to ClubConnect efficiently
+
+|`* *` |Member |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+
+|`* *` |Member |be notified on members' birthdays |wish them on time
+
+|`* *` |Member |compress the display of members |browse through the list of members with less distractions
+
+|`* *` |Efficient member |use shortcuts for commands |enter commands quickly
```

Comment:
It'll be good if some descriptions are added to the users for the other rows too